paasta_tools/__init__.py

```diff
@@ -11,9 +11,21 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+#
 # It is imperative that this file not contain any imports from our
 # dependencies. Since this file is imported from setup.py in the
 # setup phase, the dependencies may not exist on disk yet.
 #
 # Don't bump version manually. See `make release` docs in ./Makefile
+import os
+import sys
+
 __version__ = '0.35.0'
+USER_AGENT = "PaaSTA Tools %s" % __version__
+
+
+def get_user_agent():
```

Comment:
Seems way low level, why not in utils where most of our other generic functions go?